src/controllers/users.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+  const foundUser = userService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.create(name);
+
+  res.statusCode = 201;
```

Comment:
Use `res.status(201)` instead of `res.statusCode = 201` for better readability and consistency.